### PR TITLE
Added the option to confirm password

### LINKED ISSUE #270: To add an option of confirming password
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
When the user signs up through cbot , there is only one field to input password and hence many chances for error

**Expected behavior:**
To add custom password field and send data through it

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #270 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Added the option to confirm password

#### Changes proposed in this pull request:
Added the option to confirm password

### CODE REVIEW

**Comment by ms10398** [comment]
Please correct codacy check

**Comment by stealthanthrax** [comment]
@zamhaq @ms10398 @gabru-md , have a look

**Comment by stealthanthrax** [comment]
Yes @gabru-md . Thanks. Have a look now

File: src/js/signup.js [hunk 1]
```diff
@@ -54,4 +55,6 @@ signupForm.addEventListener("submit", function reset(event){
 			alert(msg);
 		}
 	});
+}
+else alert("Passwords do not match");
```
[the remark this answers is not in the document]
Comment (gabru-md) [reply src/js/signup.js]: @stealthanthrax please resolve this

**Comment by arnav-mandal1234** [comment]
Lgtm 👍

**Pull request merged**